napari/layers/labels/labels.py

```diff
@@ -551,3 +623,17 @@ def paint(self, coord, new_label, refresh=True):
 
         if refresh is True:
             self.refresh()
+
+    def get_message(self):
+        msg = super().get_message()
+        # if this labels layer has properties
+        if self._label_index and self._properties:
+            # if the cursor is not outside the image or on the background
+            if self._value is not None:
+                if self._label_index.get(self._value) is not None:
```

Comment:
```suggestion
                if self._value in self._label_index:
```

Reply:
So the reason I went with the get method is I'm not sure whether `k in dict` searches effectively, whereas I'm pretty sure `dict.get(k)` is O(1). Happy to change it though, just thought these dicts might get quite big

Reply:
Python docs confirm `k in dict` is optimised.